oolushola/Banka
# #[165169666] Unit test password reset endpoint

#### What does this PR do?
Test assertions for password reset if users forgets their password

#### Description of Task to be completed?
Have the following resource endpoints tested with tdd to check if all assertions are met.

GET: /api/v1/user/password-reset

POST: /api/v1/user/password-reset

#### How should this be manually tested?
After cloning the repo, cd into it and from the terminal RUN 'npm run test' to check the build status of the the password reset description

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#165169666](https://www.pivotaltracker.com/story/show/165169666)

#### Screenshots (if appropriate)
NA

#### Questions:
NA
